__tests__/app.test.js

```diff
@@ -148,4 +148,48 @@ describe("GET", () => {
             })
         })
     })
-});
\ No newline at end of file
+});
+
+describe('POST', () => {
+    describe('POST /api/articles/:article_id/comments', () => {
+        test('status 201 created should add a comment to comments table with appropriate properties', () => {
+            return request(app)
+            .post('/api/articles/1/comments')
+            .send( 
+                {
+                    author: 'butter_bridge',
+                    body: 'my first comment'
+                })
+            .expect(201)
+            .then(({body}) => {
+                expect(body.comment).toHaveProperty('author');
```

Comment:
we can also expect the rest of the properties that comments have automatically :)